core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4

```diff
@@ -89,9 +89,11 @@ statement
     | DROP MATERIALIZED VIEW (IF EXISTS)? qualifiedName                #dropMaterializedView
     | ALTER MATERIALIZED VIEW (IF EXISTS)? from=qualifiedName
         RENAME TO to=qualifiedName                                     #renameMaterializedView
+    | ALTER MATERIALIZED VIEW qualifiedName SET PROPERTIES properties  #setMaterializedViewProperties
```

Comment:
I think @martint mean `SET PROPERTY` here and not `SET PROPERTIES`

Reply:
Originally, he proposed `SET PROPERTY`. But I think later he agreed that it should be `SET PROPERTIES`. See
https://github.com/trinodb/trino/pull/9765#issuecomment-953117463.

Reply:
ok